File: src/main/java/me/lulu/hypixelutilities/hud/SimpleHud.java
package me.lulu.hypixelutilities.hud;

import lombok.AccessLevel;
import lombok.Getter;
import net.minecraft.client.Minecraft;
import net.minecraft.client.gui.FontRenderer;

public abstract class SimpleHud implements Hud {

    @Getter(AccessLevel.MODULE)
    private FontRenderer fontRenderer = Minecraft.getMinecraft().fontRendererObj;

}

File: src/main/java/me/lulu/hypixelutilities/hud/SimpleHud.kt
package me.lulu.hypixelutilities.hud

import lombok.AccessLevel
import lombok.Getter
import me.lulu.hypixelutilities.util.mc

abstract class SimpleHud : Hud {
    @Getter(AccessLevel.MODULE)
    private val fontRenderer = mc.fontRendererObj
}
